packages/http-client-java/emitter/src/options.ts

```diff
@@ -0,0 +1,61 @@
+import { JSONSchemaType } from "@typespec/compiler";
+
+// typespec-java has another "options.ts" file, with same "export".
+// If add/remove "export" here, please also check typespec-java in autorest.java repository.
```

Comment:
If we add/remove new configs for both branded/unbranded, do we also need to add it in autorest.java?

Reply:
It depends. If the option is not valid for Swagger (e.g. only affect TypeScript process), then no (e.g. "group-etag"?)